src/main/java/br/com/homefood/entity/Ingrediente.java

```diff
@@ -1,9 +1,19 @@
 package br.com.homefood.entity;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Ingrediente")
```

Comment:
At, nome de tabela normalmente é tudo em letra minúscula
```suggestion
@Table(name = "ingrediente")
```